catalogue/webapp/components/SingleColumnWork/SingleColumnWork.js

```diff
@@ -0,0 +1,257 @@
+// @flow
```

Comment:
nitpick `SingleColumnWorkDetails`?

Reply:
Take it back, realise it's the whole work.

Reply:
It doesn't include the Viewer, so I could change it to `SingleColumnWorkDetails`